tier4_system_msgs/msg/DiagnosticLink.msg

```diff
@@ -0,0 +1,2 @@
+uint32 index
+bool used
```

Comment:
We might want to have a README for this message to explain what these fields mean.

Reply:
Add comment https://github.com/tier4/tier4_autoware_msgs/pull/96/commits/ddcc615a2e780f93db93859644e529e732b07f72